web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -40,6 +40,7 @@
         <url-pattern>/api/v1/*</url-pattern>
         <url-pattern>/api/private/*</url-pattern>
         <url-pattern>/legacy-api</url-pattern>
+        <url-pattern>/health</url-pattern>
```

Comment:
since we'll have a client depend on this I think we should include a version number. Maybe something like `/api/internal/v1/health`?